feature/external-credential/src/test/java/com/simprints/feature/externalcredential/screens/scanocr/usecase/CaptureFrameUseCaseTest.kt

```diff
@@ -0,0 +1,34 @@
+package com.simprints.feature.externalcredential.screens.scanocr.usecase
+
+import android.graphics.Bitmap
+import androidx.camera.core.ImageInfo
+import androidx.camera.core.ImageProxy
+import com.google.common.truth.Truth.*
+import io.mockk.*
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Test
+
+internal class CaptureFrameUseCaseTest {
+
+    private lateinit var useCase: CaptureFrameUseCase
+
+    @Before
+    fun setUp() {
+        useCase = CaptureFrameUseCase()
+    }
+
+    @Test
+    fun `creates pair with correct bitmap and image info`() = runTest {
```

Comment:
Since the results are nullable, there should be at least one test where it returns null.